wowup-electron/src/app/pages/my-addons/my-addons.component.ts

```diff
@@ -780,7 +780,7 @@ export class MyAddonsComponent implements OnInit, OnDestroy, AfterViewInit {
         this._displayAddonsSrc.next(rowData);
         this.setPageContextText();
         this._cdRef.detectChanges();
-        return from(this._wowUpAddonService.persistUpdateInformationToWowUpAddon(addons));
+        return from([]);
```

Comment:
Not sure about this change, there must be a better alternative here but I'm unfamiliar with the construct used here